tests/integration/components/hyper-table-v2/manage-columns-test.ts

```diff
@@ -0,0 +1,248 @@
+  { key: 'foo', category: 'influencer', clustering_key: 'instagram' },
+  { key: 'bar', category: 'influencer', clustering_key: 'youtube' }
+];
+
+const COLUMNS = [{ key: 'code', category: 'affiliation', clustering_key: '' }];
+
+module('Integration | Component | hyper-table-v2/manage-columns', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(async function () {
+    this.tableManager = new TableManager();
+    this.rowsFetcher = new RowsFetcher();
+    this.handler = new TableHandler(this, this.tableManager, this.rowsFetcher);
+
+    sinon.stub(this.tableManager, 'fetchColumnDefinitions').callsFake(() => {
+      return Promise.resolve({
+        column_definitions: COLUMN_DEFINITIONS.reduce(
+          (columnDefinitions, column) => [
+            ...columnDefinitions,
+            ...[
+              buildColumnDefinition(column.key, {
+                clustering_key: column.clustering_key,
+                category: column.category
+              })
+            ]
+          ],
+          []
+        )
+      });
+    });
+
+    sinon.stub(this.tableManager, 'fetchColumns').callsFake(() => {
+      return Promise.resolve({
+        columns: COLUMNS.reduce(
+          (columns, column) => [
+            ...columns,
+            ...[
+              buildColumn(column.key, {
+                clustering_key: column.clustering_key,
+                category: column.category
+              })
+            ]
+          ],
+          []
+        )
+      });
+    });
+
+    await this.handler.fetchColumnDefinitions();
+    await this.handler.fetchColumns();
+  });
+
+  test('it renders', async function (assert) {
+    await render(hbs`<HyperTableV2::ManageColumns @handler={{this.handler}} />`);
+    assert.dom('.available-fields-wrapper.invisible').exists({ count: 1 });
+  });
+
+  module('when a user click on manage column button', function () {
+    test('it should open the available columns', async function (assert) {
+      await render(hbs`<HyperTableV2::ManageColumns @handler={{this.handler}} />`);
+      await click('.upf-btn.upf-btn--primary');
+
+      assert.dom('.available-fields-wrapper.visible').exists({ count: 1 });
+    });
+
+    module('when user manage categories', function () {
+      test('it displays all categories', async function (assert) {
+        await render(hbs`<HyperTableV2::ManageColumns @handler={{this.handler}} />`);
+        await click('.upf-btn.upf-btn--primary');
+
+        assert.dom('.available-fields-wrapper__categories').exists({ count: 1 });
+        assert.dom('[data-control-name="field_category_toggle_all_fields"]').exists({ count: 1 });
+        assert.dom('[data-control-name="field_category_toggle_affiliation"]').exists({ count: 1 });
+        assert.dom('[data-control-name="field_category_toggle_influencer"]').exists({ count: 1 });
+      });
+
+      test('it displays default category as active', async function (assert) {
```

Comment:
`it displays the default category as active`

Reply:
line 96: `it displays all the column definitions...`
line 114: `it sets the category...`
line 123: `it displays the column definitions...`

Reply:
ty